tests/integration/worker/test_remote_submission_worker.py

```diff
@@ -0,0 +1,44 @@
+import mock
+import os
+import responses
+import unittest
+
+import scripts.workers.remote_submission_worker as remote_submission_worker
+
+
+class TestDownloadAndExtractFile(unittest.TestCase):
+    def setUp(self):
+        self.download_dir = "/tmp/evalai/"
+        self.download_filename = "file"
+        self.download_path = "{}{}".format(self.download_dir, self.download_filename)
+
+        self.url = "http://example.com"
+        self.body = """{"content": "test-data"}"""
+        responses.add(
+            responses.GET,
+            self.url,
+            body=self.body,
+            status=200
+        )
+
+        try:
+            os.makedirs(self.download_dir)
+        except OSError:
+            pass
```

Comment:
@takitsuse can you add a log statement here, in case directory creation fails we won't know what exactly the issue is

Reply:
Sure! I am planning to mock the logger and exec `logger.error()` in except statement and exec `mock_logger.assert_called()` in the test function.
Does this sounds okay to you?

Reply:
Hmm...I looked around other testing codes in EvalAI and I found no examples of logging statement is being used for creating directory even though `os.makedirs()` I found ocassionally.
Do I really have to add logging statement for creating directory?

Reply:
@takitsuse instead of writing your own `mkdir` can you use this https://github.com/Cloud-CV/EvalAI/blob/master/scripts/workers/remote_submission_worker.py#L146. In case this fails you can your mock_logger to log the error